components/Head.js

```diff
@@ -7,6 +7,7 @@ export default class extends React.Component {
       <Head>
         <meta name='viewport' content='width=device-width, initial-scale=1' />
         <link rel='manifest' href='static/manifest.json' />
+        <link href='https://fonts.googleapis.com/css?family=Open+Sans|Roboto' rel='stylesheet' />
         <title>Resistance Calendar</title>
```

Comment:
Eventually it would be awesome to make these fonts local, as calling the google fonts remotely can really slow things down. If it's a quick thing, then maybe go for it, if not, maybe make a To Do to download those and host them ourselves.